src/components/Login.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-container fluid fill-height class="pa-0 ma-0">
+    <v-layout align-center justify-center>
+      <v-flex text-xs-center fill-height>
+        <v-card dark tile
+                color="#36363f"
+                height="100%"
+          >
+          <v-toolbar dark flat color="#474755">
+            <v-toolbar-title> Login </v-toolbar-title>
+          </v-toolbar>
+          <v-icon size="196px" class="pt-5"> mdi-account-circle</v-icon>
+          <div :class="{'margin-large': $vuetify.breakpoint.lgAndUp,
```

Comment:
Pls don't create ur own classes when there already exists breakpoints. Use the grid system effectively

Reply:
done